scarb/src/bin/scarb/args.rs

```diff
@@ -135,6 +135,8 @@ pub enum Command {
     /// Output the resolved dependencies of a package, the concrete used versions including
     /// overrides, in machine-readable format.
     Metadata(MetadataArgs),
+    // Resolve the workspace dependencies by downloading it
```

Comment:
```suggestion
    // Fetch dependencies of packages from the network.
```

nitpick: this suggestion explicitly says about network effect, which is what this command is about